src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiEnchantColour.java

```diff
@@ -192,8 +196,32 @@ public void drawScreen(int mouseX, int mouseY, float partialTicks) {
         GlStateManager.color(1, 1, 1, 1);
         Minecraft.getMinecraft().getTextureManager().bindTexture(help);
         Utils.drawTexturedRect(guiLeft + xSize + 3, guiTopSidebar - 18, 16, 16, GL11.GL_NEAREST);
+        if(maxedBookFound == 0){
+            try {
+                if (NotEnoughUpdates.INSTANCE.manager.jsonToStack(
+                        NotEnoughUpdates.INSTANCE.manager.getItemInformation().get("MAXED_ENCHANT_BOOK")).hasDisplayName()) {
+                    maxedBook = NotEnoughUpdates.INSTANCE.manager.jsonToStack(NotEnoughUpdates.INSTANCE.manager.getItemInformation().get("MAXED_ENCHANT_BOOK"));
+                    maxedBookFound = 1;
+                } else {
+                    maxedBookFound = 2;
+                }
+                //TODO: Actually write something to catch any errors
```

Comment:
this should be done before I merge the pr

Reply:
i prob should of removed that comment but I made it not show if the item isn't loaded https://canary.discord.com/channels/516977525906341928/896407218151366687/912649488886296586